Update hset.md (#2234)
redis/redis-doc: commands/hset.md

Sets the specified fields to their respective values in the hash stored at `key`.

> This command overwrites any specified fields already existing in the hash. If `key` does not exist, a new key holding a hash is created.

Reviewer: ```suggestion
This command overwrites any specified fields that already exist in the hash.
If `key` doesn't exist, a new key that stores the hash is created.
```

> This command overwrites any specified fields already existing in the hash. If `key` does not exist, a new key holding a hash is created.

Reviewer: ```suggestion
This command overwrites the values of specified fields that exist in the hash.
If `key` doesn't exist, a new key holding a hash is created.
```

> ```cli

Reviewer: I don't understand the added value of this change to the example.

Author: `hset` can add multiple values. but the original explanation and example was showing only 1 field.
I wanted to demonstrate this command can add single and multiple fields without checking `hmset` command and its deprecation note.

HSET myhash field1 "Hello"
HGET myhash field1

> HMSET myhash field2 "Hi" field3 "World"

Reviewer: `HMSET` is considered deprecated.

```suggestion
HSET myhash field2 "Hi" field3 "World"
```

Author: Correct, you made it as I wanted to be.

> HMSET myhash field2 "Hi" field3 "World"

Reviewer: ```suggestion
HSET myhash field2 "Hi" field3 "World"
```
